### PR TITLE
Api call util

### PR DESCRIPTION
**Pull request by pdeleastar**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,40 +4,69 @@ import News from './components/News'
...
 import Price from './components/Price'
 import Error from './components/Error'
-import Search  from './components/Search.jsx';
+import Search from './components/Search.jsx'
 import { connect } from 'react-redux'
+import * as logo from './images/logo.png'
 
 class App extends Component {
   render () {
     return (
```
Comment (snony) [inline src/App.js]: Perhaps, no need for the return statement here

File: src/App.js [hunk 2]
```diff
@@ -4,40 +4,69 @@ import News from './components/News'
...
-      <div>
-        <div>
-         <Search />
-          <Price price={this.props.price} />
-          <Error error={this.props.error} />
-        </div>
+      <div className='gradient'>
```
Comment (snony) [inline src/App.js]: There is a lot of nesting, would it be possible to extract these divs into their respective components?

File: src/actions/actions.js [hunk 3]
```diff
@@ -8,6 +10,7 @@ export const PEER_ACTION = 'PEER ACTION'
 export const MARKET_ACTION = 'MARKET ACTION'
 export const PRICE_ACTION = 'PRICE ACTION'
 export const ERROR_ACTION = 'ERROR ACTION'
+export const SYMBOLS_ACTION = 'SYMBOLS ACTION'
 
 function keyStatAction (payload) {
```
Comment (snony) [inline src/actions/actions.js]: Perhaps remove functions that returns nothing

File: src/components/KeyStats.jsx [hunk 4]
```diff
@@ -6,22 +6,42 @@ const KeyStats = props => {
...
       {keystats && (
-        <div>
-          <h3>KEY STATS</h3>
-          <hr />
+        <div className="keystats">
+          <div className="category-header">
+            <h3>KEY STATS</h3>
+          </div>
+
           <div className="keystats-row">
```
Comment (TobyLi96) [inline src/components/KeyStats.jsx]: Could use [CSS grid](https://css-tricks.com/snippets/css/complete-guide-grid/) to organize each key stats. Helps avoid the nesting.

File: src/actions/actions.js [hunk 5]
```diff
@@ -65,52 +68,41 @@ function priceAction (payload) {
...
+    apiCall('news', input, newsAction, dispatch)
+    apiCall('chart', input, chartAction, dispatch)
+    apiCall('peer', input, peerAction, dispatch)
+    apiCall('keystats', input, keyStatAction, dispatch)
+    apiCall('price', input, priceAction, dispatch)
+    apiCall('market', input, marketAction, dispatch)
   }
 }
 
 export function chartFilter (input, time) {
```
Comment (snony) [inline src/actions/actions.js]: Aim for modularity. As such, I would suggest to extract these different actions that are associated to their respective components folder such as news/action.js etc..

File: src/components/Overview.jsx [hunk 6]
```diff
@@ -6,12 +6,15 @@ const Overview = props => {
...
-          <hr />
-          <h2>{overview.companyName}</h2>
-          <p>{overview.website}</p>
-          <p>{overview.description}</p>
+        <div className="overview">
+          <div className="category-header">
+            <h3>COMPANY OVERVIEW</h3>
+          </div>
+          <div className="overview-content">
+            <h2>{overview.companyName}</h2>
```
Comment (TobyLi96) [inline src/components/Overview.jsx]:
```
const { companyName, website, description } = props.overview
```

File: src/components/SymbolInformation.jsx [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const SymbolInformation = props => {
```
Comment (snony) [inline src/components/SymbolInformation.jsx]: Why is Symbolinformation function returning nothing?

Comment (pdeleastar) [reply src/components/SymbolInformation.jsx]: Added content here
https://github.com/pdeleastar/stock-trader/pull/10/files#diff-5c25524bbdf95fa3d0a99cbf04cb1e0b

**Pull request merged**